src/parser.ts

```diff
@@ -92,6 +97,21 @@ export class TrailingCommaError implements SourceError {
   }
 }
 
+export function parseAt(source: string, num: number) {
+  let program: es.Program | undefined
+  try {
+    program = (acornParseAt(source, num) as unknown) as es.Program
+    ancestor(program as es.Node, walkers, undefined)
```

Comment:
Why does this need to use the ancestor function?

I thought it just needed to get the first string literal.